vanilla-framework/vanilla-framework
# Remove offset from focus state

## Done
- Updated focus state based on design discussion [here](https://github.com/ubuntudesign/vanilla-design/issues/331#issuecomment-433897488)
- Removed offset so focus state sits on top of component

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check Buttons, Forms including validation

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/2041

### Visual
![accessible-focus-state-001](https://user-images.githubusercontent.com/17748020/47649122-23e99500-db74-11e8-8724-1937069caa5a.png)
